app/src/main/java/com/ataulm/stacks/NavHostActivity.kt

```diff
@@ -0,0 +1,5 @@
+package com.ataulm.stacks
+
+import androidx.appcompat.app.AppCompatActivity
+
+class NavHostActivity : AppCompatActivity(R.layout.activity_nav_host)
```

Comment:
At this time, there isn't really any top-level navigation so this activity's sole purpose is to hold the NavHostFragment